### DISCUSSION TITLE
WebRadio links

### DISCUSSION THREAD

**Discussion by madcgm** [rern/rAudio#795] [Feature Requests]
It would be nice if you supply a nice list of web-radio links out of the box.
If they are categorized by genre and/or country, much better.
Thanks in advance.

**Comment by scassidy** [comment]
It would be a pain to maintain!

I recently found this list, which seems to be very reliable: http://fmstream.org/index.php

**Comment by hjfisch** [comment]
Take a look at http://radiobit.50webs.com

**Comment by hjfisch** [comment]
[Internet Radio.zip](https://github.com/rern/rAudio-1/files/8036615/Internet.Radio.zip)
Some Jazz Stations incl. Logos.

**Comment by ghost** [comment]
some more: https://hendrikjansen.nl/henk/streaming.html
